build_tools/benchmarks/export_benchmark_config.py

```diff
@@ -200,6 +216,15 @@ def _parse_benchmark_presets(
   return matchers
 
 
+def _parse_shard_count(arg: str):
```

Comment:
Parsing here seems to be a little bit tricky. I'm wondering if we can just ask users to provide `default=x` if they want to shard all targets with the same number. So

1. No `--shard_count` -> No sharding
2. `--shard_count="{default=10}"` -> Shard all targets with 10
3. `--shard_count="{c2-standard-16=2}"` -> Only shard `c2-standard-16`
4. `--shard_count="{c2-standard-16=2,default=10}"` -> Shard `c2-standard-16` with 2, other with 10

Reply:
Yep. Changed.